ptvsd/wrapper.py

```diff
@@ -1988,16 +1987,19 @@ def on_pause(self, request, args):
                 return
 
         # Always suspend all threads.
+        # TODO: Replace this with suspend all command after #732 is fixed
         for pyd_tid in self.thread_map.pydevd_ids():
             self.pydevd_notify(pydevd_comm.CMD_THREAD_SUSPEND, pyd_tid)
         self.send_response(request)
 
     @async_handler
     def on_continue(self, request, args):
         # TODO: docstring
-        tid = self.thread_map.to_pydevd(int(args['threadId']))
-        self.pydevd_notify(pydevd_comm.CMD_THREAD_RUN, tid)
-        self.send_response(request)
+        # Always continue all threads.
+        # TODO: Replace this with resume all command after #732 is fixed
+        for pyd_tid in self.thread_map.pydevd_ids():
+            self.pydevd_notify(pydevd_comm.CMD_THREAD_RUN, pyd_tid)
+        self.send_response(request, allThreadsContinued=True)
```

Comment:
Can't we have a single command that would tell pydevd to continue ALL threads. Feels a lot more efficient, and code world work better. 

Also should be done for suspend all threads also If not already done. I.e. when we got the issue button we should send a single request to pause ALL threads instead of sending a request for each these.

Reply:
For instance pause will fail if a new thread is created in the interim (a race condition resulting in a bug). So technically it should be pydeb that does the necessary plumbing and iteration of threads.

Reply:
I have already discussed this with @fabioz see here https://github.com/Microsoft/ptvsd/issues/732. :)